mms/model_service_worker.py

```diff
@@ -433,6 +438,25 @@ def run_server(self):
             raise
 
 
+def emit_metrics(metrics):
```

Comment:
Should this be a part of this file? Should we have it as a part of a separate metric file?

Reply:
as it is a single function, I do not see a strong reason for this to be moved, the emission is specific to the worker, hence I left it here.